portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 90%;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
 }
+a:link {
```

Comment:
Another minor point: we usually include a blank line between different rules, https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation